core/ui/src/commonMain/kotlin/org/mifospay/core/ui/PasswordStrengthIndicator.kt

```diff
@@ -89,10 +88,10 @@ fun CombinedPasswordErrorCard(
         PasswordStrengthState.NONE -> MaterialTheme.colorScheme.error
         PasswordStrengthState.WEAK_1 -> MaterialTheme.colorScheme.error
         PasswordStrengthState.WEAK_2 -> MaterialTheme.colorScheme.error
-        PasswordStrengthState.WEAK_3 -> weakColor
+        PasswordStrengthState.WEAK_3 -> MaterialTheme.colorScheme.error
         PasswordStrengthState.GOOD -> MaterialTheme.colorScheme.primary
-        PasswordStrengthState.STRONG -> strongColor
-        PasswordStrengthState.VERY_STRONG -> Color.Magenta
+        PasswordStrengthState.STRONG -> MaterialTheme.colorScheme.primary
```

Comment:
the `Good` and `Strong` and `VeryStrong` should have two different color. cause we want to show user different color when these takes one of these states we don't want to show the same color for both `Good` and `Strong`